particles/Layout/source/DetailSlider.js

```diff
@@ -8,8 +8,9 @@
  * http://polymer.github.io/PATENTS.txt
  */
 
+/* global defineParticle */
```

Comment:
nit - is this comment needed (here and everywhere)?

Reply:
It tells editors/linters not to complain about undeclared variables.